### PR TITLE
[doc] Use related path for img

### LINKED ISSUE #9426: [Bug] [Doc] Image can not display properly
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

Due to the migration of the documents directly from [website](https://github.com/apache/dolphinscheduler-website), some of the file paths were incorrect, thus preventing the images from being displayed properly.

### What you expected to happen

Modify the path so as to ensure that the images can be displayed properly.

### How to reproduce

You can see the problem at this [link](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/zh/guide/task/datax.md). The situation is similar to the other documents.

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhongjiajie**
Use related path in our docs for imgs, previous
can not use because website need the absolute path
from root directory, and after we merged
apache/dolphinscheduler-website#789 we have covert
function to do that

close: #9426

### CODE REVIEW

**Comment by zhongjiajie** [comment]
PTAL @QuakeWang

File: docs/docs/en/architecture/design.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ### System Architecture Diagram
 
 <p align="center">
-  <img src="/img/architecture-1.3.0.jpg" alt="System architecture diagram"  width="70%" />
+  <img src="../../../img/architecture-1.3.0.jpg" alt="System architecture diagram"  width="70%" />
```
Comment (QuakeWang) [inline docs/docs/en/architecture/design.md:8]: If possible, we can use markdown instead of html : )

**Review by QuakeWang — COMMENTED** [review]
The image formatting part of the document contains HTML and it is recommended that this be changed to MarkDown format.

File: docs/docs/en/architecture/design.md (hunk 1 above)
Comment (zhongjiajie) [reply docs/docs/en/architecture/design.md:8]: I think it is out of this PR or issue scope, and we already have a some related issue in https://github.com/apache/dolphinscheduler/issues/8391, and we can fix this issue in that PR. WDTY?

**Review by QuakeWang — APPROVED** [review]
LGTM

File: docs/docs/en/architecture/design.md (hunk 1 above)
Comment (QuakeWang) [reply docs/docs/en/architecture/design.md:8]: > I think it is out of this PR or issue scope, and we already have a some related issue in #8391, and we can fix this issue in that PR. WDTY?

Ok, I have approved it!

**Review by SbloodyS — APPROVED** [review]
+1

**Comment by zhongjiajie** [comment]
I add milestones to 3.0.0-alpha which will be directly release from dev branch

**Pull request merged**